README.md

```diff
@@ -179,6 +179,28 @@ This will set the ClientId and ClientSecret, which should be taken from the 'app
   "ConnectionString" : "..."
 }
 ```
+#### Mailer.
+
+To set up the mailer, it is only necessary to add the authentication information into the environment variables. If you choose to do so via the appsetings.Development.json file, the mailer config section should look something like this:
```

Comment:
i would put "To do this via..." instead of "If you choose to do so via.." What do you think?